### ISSUE TITLE
CRL support

### ISSUE THREAD

**Issue by dbarrosop** [CZ-NIC/jetconf#7]
Hello,
does jetconf support checking a CRL? Couldn't find anything.

Regards
David

**Comment by alesmrazek** [comment]
Hello, 
we have not dealt with CRL yet. [pyssl doc](https://docs.python.org/3/library/ssl.html?highlight=sslcertverificationerror#ssl.VERIFY_DEFAULT) and [lines](https://github.com/CZ-NIC/jetconf/blob/5534fc65d7f653811523bfdfd6083990d2df16ae/jetconf/rest_server.py#L289) in `restconf_server.py` module might be helpful. What exactly do you want to do? 

Aleš

**Comment by dbarrosop** [comment]
Unless I am missing something if you revoke a certificate jetconf will still treat it as a valid one. That means you can't disable users easily. I guess you could use NACM but that seems to be a bit overkill for simpler scenarios where you don't need fine-grain control. And even with NACM, it's a proven first line of defense in case of NACM misconfiguration.

**Comment by alesmrazek** [comment]
You are right. I am trying to test build in [pyssl](https://docs.python.org/3/library/ssl.html?highlight=sslcertverificationerror#ssl.VERIFY_CRL_CHECK_LEAF) implementation. `ssl.VERIFY_CRL_CHECK_CHAIN` or `ssl.VERIFY_CRL_CHECK_LEAF` values of `SSLContext.verify_flags` should do CRL checking. What format of CRL do you use?
